### DISCUSSION TITLE
post_233821

### DISCUSSION THREAD

**Discussion by giscus** [cincuranet/blog-comments#1354] [Comments]
# post_233821

https://www.tabsoverspaces.com/233821-firebird-database-with-more-than-2-32-transactions

**Comment by aeadbhard** [comment]
Thank you, Jiří! Do you happen to have a version of this database but with ODS 11.2? I would like to make some test with a db with TX greater than 2^32 but I must use Firebird 2.5x. Thanks!

**Reply by cincuranet** [reply]
I don't.
